TestProject/output/GIFOutput.php

```diff
@@ -69,4 +75,47 @@ function finishOutput()
         }
     }
 
+    /**
+     * Defines output-specific parameters, which are required when calling one of the following options ->
+     * \"gifCellSize\ or \"gifCellColor\"
+     *
+     * @param Getopt $_options Defines output-specific parameters
+     */
+    function startOutput(Getopt $_options)
+    {
+        if ($_options->getOption("gifCellSize"))
+        {
+            $this->cellSize = intval($_options->getOption("gifCellSize"));
+
+            if ($this->cellSize <= 0)
+                $this->cellSize = 5;
+        }
+
+        if ($_options->getOption("gifCellColor"))
+        {
+            $this->gifCellColor = explode(",", $_options->getOption("gifCellColor"));
+        }
+    }
+
+
+    /**
+     *\"gifCellSize\" set the size for each cell and needs to be defined in pixel format -> 10 for example.
```

Comment:
Please add a summary sentence to the documentation. Something like `Adds gif-specific output options.`
Then you can explain the different options this output adds (as you already did)